applications/fax/src/fax_monitor.erl

```diff
@@ -70,6 +71,9 @@ handle_call(_Request, _From, State) ->
 %% @end
 %%------------------------------------------------------------------------------
 -spec handle_cast(any(), state()) -> kz_types:handle_cast_ret_state(state()).
+handle_cast('migrate_pending', State) ->
+    fax_maintenance:migrate_pending_faxes(),
```

Comment:
@swysor this will most likely crash for large number of pending faxes. lets remove it and admins should use `sup fax_maintenance migrate_pending_faxes`